File: PRN211/Demo_PRN211_SE1632/WebApplication1/Models/Student.cs
namespace WebApplication1.Models
{
    public class Student
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public int Mark { get; set; }

        public Student(string code, string name, int mark)
        {
            Code = code;
            Name = name;
            Mark = mark;
        }

        public override string ToString()
        {
            return $"Code: {Code}, Name: {Name}, Mark: {Mark}";
        }
    }
}

File: PRO192/Workshop6/src/Student.java
public class Student {
    
    private String code;
    private String name;
    private int mark;

    public Student(String code, String name, int mark) {
        this.code = code;
        this.name = name;
        this.mark = mark;
    }

    public String getCode() {
        return code;
    }

    public String getName() {
        return name;
    }

    public int getMark() {
        return mark;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setMark(int mark) {
        this.mark = mark;
    }

    @Override
    public String toString() {
        return String.format("%-10s%-20s%-10d", getCode(), getName(), getMark());
    }
}
